float64 step_size
int64 step_num
float64 velocity
int64 wait
---
int8 return_code
